dato.config.js

```diff
@@ -68,6 +68,11 @@ function transformItem(item) {
     $('img').remove()
     item.body = $('body').html()
   }
+  else if (item.type === 'video') {
+    if (item.video && item.title) {
+      item.video.title = item.title
+    }
```

Comment:
It also needs the values for autoplay, loop and mute from Dato. As it stands now, the toggles, they do nothing 

https://www.google.nl/search?q=the+goggles+they+do+nothing